coordinator/internal/transitengineapi/transitengineapi.go

```diff
@@ -155,6 +223,15 @@ func writeJSONResponse(w http.ResponseWriter, payload any) error {
 	return json.NewEncoder(w).Encode(response)
 }
 
+// writehttpError sends the httpError handed in as json body error response and logs prior.
+func writehttpError(w http.ResponseWriter, httpErr httpError, logger *slog.Logger) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(httpErr.code)
+	logger.Error("HTTP error", slog.Any("error", httpErr))
```

Comment:
Is this different from just doing 

```suggestion
	logger.Error("HTTP error", "error", httpErr)
```